at_secondary/at_secondary_server/lib/src/verb/handler/update_verb_handler.dart

```diff
@@ -64,11 +64,19 @@ class UpdateVerbHandler extends ChangeVerbHandler {
     // Sets Response bean to the response bean in ChangeVerbHandler
     await super.processVerb(response, verbParams, atConnection);
     var updateParams = _getUpdateParams(verbParams);
-
+    if (updateParams.sharedBy != null &&
+        updateParams.sharedBy!.isNotEmpty &&
+        AtUtils.formatAtSign(updateParams.sharedBy) !=
+            AtSecondaryServerImpl.getInstance().currentAtSign) {
+      logger.warning(
+          'Invalid update command sharedBy atsign ${AtUtils.formatAtSign(updateParams.sharedBy)} should be same as current atsign ${AtSecondaryServerImpl.getInstance().currentAtSign} ');
+      throw InvalidAtKeyException(
+          'SharedBy atsign should be the same as current atsign');
```

Comment:
Maybe make the exception message the same as what is logged above - i.e. include the actual sharedBy and current atSigns

Reply:
done